ui/src/pages/plan/rules/formUtils.test.ts

```diff
@@ -25,16 +25,14 @@ export function setDayOfMonth(formElement: any, day: number) {
     fireEvent.change(dayOfMonthInput, { target: { value: String(day) }});
 }
 
-export function setMonthOfYear(formElement: any, month: number) {
-    const dayOfMonthInput = formElement.getByLabelText(/Month of Year/i);
-    expect(dayOfMonthInput).toBeInTheDocument();
-    fireEvent.change(dayOfMonthInput, { target: { value: String(month) }});
-}
-
 export function setDaysOfWeek(formElement: any, daysOfWeek: string[]) {
-    const dayOfWeekSelect = formElement.getByLabelText(/Days of Week/i);
-    expect(dayOfWeekSelect).toBeInTheDocument();
-    userEvent.selectOptions(dayOfWeekSelect, daysOfWeek);
+    const buttons: HTMLButtonElement[] = Array.from(formElement.container.querySelectorAll("[data-testid=dayofweekcontrol] > button"));
```

Comment:
I switched away from `<select multiple>` - harder to style - to a list of toggleable buttons. Much nicer.